tests/test_integration.py

```diff
@@ -0,0 +1,151 @@
+  
+## Importing these packages is specific for this simulation case
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import xarray as xr
+from mne.io import read_info
+from scipy.stats import gamma
+from pathlib import Path
+
+import hmp
+from hmp import simulations
+
+epoch_data_file = Path("tutorials", "sample_data", "sample_data.nc")
+info_data_file = Path("tutorials", "sample_data", "eeg", "processed_0022_epo.fif")
+
+def test_integration():
+    fig, ax = plt.subplots(1)#captures plots
+    cpus = 1 # For multiprocessing, usually a good idea to use multiple CPUs as long as you have enough RAM
```

Comment:
I rather think the testing code should contain cpus = 1 and cpus = 2 as this calls different parts of the code

Reply:
Yes, so that we can do with `mark.parametrize` I can set it up so you can see how it works.

Reply:
Oh I saw that too late. If you look at commit [1468793](https://github.com/GWeindel/hmp/pull/153/commits/14687935356975d5e6249e380c8a5d457f5570db)

It's only really needed in two places, I added them. Do you think ```mark.parametrize``` is still useful in this case?

Reply:
It depends. If n_cpu == 2 works implies that n_cpu == 1 works, then I don't think it is necessary. But if they use different code paths, it might still be useful. Of course this test will be twice as long (but it's not that slow at the moment).